testing/kuttl/e2e/scaledown/01-errors.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: scaledown
+status:
+  instances:
+    - name: instance1
```

Comment:
I added the assert & the errors files because I liked the explicitness of saying, for instance, "we want to find this pod, but we don't want this other pod to exist"; but now that I'm looking at it again, I think anything I wanted to be explicit about is already captured by the asserts on the cluster (i.e., the `status.instances` asserts).